build.gradle

```diff
@@ -38,6 +38,7 @@ plugins {
 
 // Establish version and status
 ext.githubProjectName = rootProject.name // Change if github project name is not the same as the root project's name
+version = '3.15.0-FS'
```

Comment:
What version is this ? Can we add more details in the variable names ?

Reply:
Since I am modifying the conductor code the jar created with this 3.15.0 version of conductor code I have added "-FS" suffix to it to denote used for freshservice.